environment.yml

```diff
@@ -0,0 +1,11 @@
+name: egu22pygmt
+channels:
+  - conda-forge
+  - nodefaults
+dependencies:
+  - gmt=6.3.0
```

Comment:
Is this just for reproducibility (i.e., make sure gmt is at 6.3)? Otherwise shouldn't it be installed as a dependency for pygmt?

Reply:
Yes this is for reproducibility (ensure that things don't break in >1 year). I'll make sure to create a locked environment following https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#building-identical-conda-environments or with [`conda-lock`](https://github.com/conda-incubator/conda-lock) once we've finalized the tutorials.